lib/Importer/caffe2.proto

```diff
@@ -126,6 +126,10 @@ message QTensorProto {
   repeated int32 data = 6 [packed = true];
   optional string name = 7;
   optional TensorProto.DataType data_type = 8 [default = INT32];
+  repeated double scales = 9;
```

Comment:
Can we remove `scale` and `bias` now?

Reply:
@jackm321  No we cant... Remove things from proto is not a good idea since we dont know who  else is using it.